components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDO.java

```diff
@@ -211,12 +218,48 @@ public void setRefreshTokenExpiryTime(long refreshTokenExpiryTime) {
         this.refreshTokenExpiryTime = refreshTokenExpiryTime;
     }
 
+    /**
+     * @deprecated use {@link #getIdTokenAudiences()} instead.
```

Comment:
and access token audience method too right?

Reply:
fixed with 95271dc3c7cc820ee8b77a92ce75c0129db0015b